cfgov/ask_cfpb/scripts/export_ask_data.py

```diff
@@ -99,51 +75,38 @@ def assemble_output(spanish_only=False):
     return output_rows
 
 
-def export_questions(spanish_only=False):
+def export_questions(path=None):
     """
-    A rather ridiculous script for exporting Ask CFPB Answer content.
+    A script for exporting Ask CFPB Answer content
+    to a CSV that can be opened easily in Excel.
 
     Run from within cfgov-refresh with:
     `python cfgov/manage.py runscript export_ask_data`
 
     CEE staffers use a version of Excel that can't easily import UTF-8
-    non-ascii encodings. Generally the only Ask content that has characters
-    outside the ascii range is Spanish answers, so we export the bulk of the
-    data as UTF-8, and Spanish answers as a separate UTF-16 file that our
-    versions of Excel will open with proper diacritical marks.
-    UTF-16 doubles the file size, which can cause Excel performance issues
-    for the full data set. UTF-16 also plays hob with delimiting,
-    pushing text to next cells and making a mess.
-
-    So passing `--script-args spanish` will output just the Ask IDs, URLs,
-    and Spanish answers in UTF-16 to a well-behaved tab-separated file.
-    The default (no script args) outputs the full data set to CSV,
-    minus Spanish answers, in UTF-8.
+    non-ascii encodings. So we throw in the towel and encode the CSV
+    with windows-1252.
+
+    The script will dump the file to `/tmp/` unless a path argument
+    is supplied. A command that passes in path would look like this:
+    `python cfgov/manage.py runscript export_ask_data --script-args [PATH]`
     """
 
     timestamp = datetime.datetime.now().strftime("%Y-%m-%d")
-    if spanish_only:
-        filename = 'spanish_answers_{}.tsv'
-    else:
-        filename = 'answers_{}.csv'
-    with open(filename.format(timestamp), 'w') as f:
-        if spanish_only:
-            f.write(
-                "\t".encode('UTF16').join(
-                    SPANISH_HEADINGS) + '\n'.encode('UTF16'))
-            for row in assemble_output(spanish_only=True):
-                f.write("\t".encode('UTF16').join(
-                    [row[key] for key
-                     in SPANISH_HEADINGS]) + '\n'.encode('UTF16'))
-        else:
-            writer = csvkit.UnicodeWriter(f)
-            writer.writerow(HEADINGS)
-            for row in assemble_output():
-                writer.writerow([row[key] for key in HEADINGS])
+    slug = 'ask-cfpb-{}.csv'.format(timestamp)
+    if path is None:
```

Comment:
What do you think about setting the default in the definition, e.g. `def export_questions(path='/tmp')` ?